src/dapp/libexec/dapp/dapp-verify-contract

```diff
@@ -28,14 +28,17 @@ case "$chain" in
     exit 1
 esac
 
-meta=$(cat "$DAPP_OUT"/"${1?contractname}"_meta.json)
+path=${1?contractname}
```

Comment:
What are the changes in this file doing?

Reply:
According to @nanexcool this is broken anyway. This is just a best effort try to update it to this new build format. It should be properly fixed in another pr.